src/isar_turtlebot/robotinterface.py

```diff
@@ -38,22 +37,71 @@ def __init__(self):
 
         self.bridge: RosBridge = RosBridge()
 
+        self.inspection_task_timout: int = config.get(
```

Comment:
Spelling mistake. Timeouten should be a float. Need to either use config.getint or config.getfloat

Reply:
Use the timeout variable in the wait_for_task function thingy